charts/seed-bootstrap/charts/fluent-bit/templates/_helpers.tpl

```diff
@@ -124,6 +124,19 @@ filter-kubernetes.conf: |-
       Parser              lokiCuratorParser
       Reserve_Data        True
 
+{{ if .Values.eventLogger.enabled }}
+  [FILTER]
+      Name                nest
+      Match               kubernetes.*event-logger*event-logger*
+      Operation           lift
+      Nested_under        log
+
+  [FILTER]
+      Name record_modifier
+      Match kubernetes.*event-logger*event-logger*
+      Record job event-logging
+{{- end }}
```

Comment:
Could you place the event-logger into into a sub package of `operation/botanist/component/logging` and move the logging configuration into the new package like shown in [this example](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/component/kubescheduler/logging.go)?